blueprints.yaml

```diff
@@ -116,6 +116,21 @@ form:
                   help: PLUGIN_GANALYTICS.BLOCKED_IPS_HELP
                   value_only: true
 
+                blockedIpRanges:
+                  type: array
```

Comment:
Please use [mutliple select](https://learn.getgrav.org/forms/forms/fields-available#the-select-field) as field type. For you and the user it is easier to use. :+1: